client/frontend.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"log"
 
+	pbc "github.com/opiproject/opi-api/common/v1/gen/go"
 	pb "github.com/opiproject/opi-api/storage/v1/gen/go"
 	"google.golang.org/grpc"
 )
```

Comment:
looks good. optionally we can use `uuid.New().String()` for the `ObjectKey` just to keep things random and interesting... 
not must for this PR

Reply:
I kept them somewhat morel legible for easy debugging, let's discuss...